docs/content/en/latest/learn/embed_dashboards/web_components/ai_chat_custom_element.md

```diff
@@ -0,0 +1,56 @@
+---
+title: AI Chat custom element
+sidebar_label: AI Chat custom element
+copyright: (C) 2007-2025 GoodData Corporation
+id: webcomponents_aichat
+---
+
+You can embed the GoodData AI Chat into your application using the `gd-ai-chat` custom element with the following:
+
+```html
+<!-- minimal setup -->
+<gd-ai-assistant></gd-ai-assistant>
+```
+
+## Supported attributes
+
+```html
+<!-- all supported attributes -->
+<gd-ai-assistant workspace="my-workspace-id" locale="en-US"></gd-ai-assistant>
+```
+
+-   `workspace` - optional, the ID of the workspace for this AI Chat. By default, it is taken from the context (e.g., from the script URL).
+-   `locale` - optional, defaults to `en-US`. The localization of the AI Chat interface. For all available languages, see [the full list of available localizations][1].
+
+You can also provide a workspace id on the context level instead of passing it as an attribute to every AI Chat component. See the code example of the [Web Components authentication][2] article.
+
+> The locale property affects only the UI elements and not your data or the metadata language.
+
+## Supported events
+
+`gd-ai-assistant` emits the following events:
+
+-   `linkClick` - when a user clicks on a link in the chat.
+-   `chatOpened` - when the chat is opened.
```

Comment:
```suggestion
```

Reply:
Since window is not part of the component, this event will never be called and does not make sense